Translator/Src/Reader.cpp

```diff
@@ -0,0 +1,65 @@
+#include "Reader.h"
+#include <fstream>
+#include <sstream>
+#include <iostream>
+
+Reader::Reader() {}
+
+Reader::~Reader() {}
+
+// Convert a point to a string for map lookup (e.g., "x,y,z")
+std::string Reader::pointToString(const Point& pt) const {
+    std::ostringstream oss;
+    oss << pt.getX() << "," << pt.getY() << "," << pt.getZ();
```

Comment:
what is the use of commas as in our stl file there is no comas, then how can comma make a difference between two coordinates.